andela/-Healthchecks_BauMeister-
# #141392695 Configure timeouts and grace periods to more than 30 days.

#### What does this PR do?
Have users set timeout and grace periods on Cron jobs for more than 30 days.
#### Description of Task to be completed?
- Change noUiSlider to extended grace period to more than a 30 day period.
- Enable calculation of months from the time set by the user.
- Change maximum value for the grace period.
#### How should this be manually tested?
- After logging into healthchecks a user should be able to configure and save grace periods and timeouts for more than 30 days.
#### What are the relevant pivotal tracker stories?
#141392695
